gui-apps/grim/grim-1.4.0-r1.ebuild

```diff
@@ -10,10 +10,11 @@ HOMEPAGE="https://github.com/emersion/grim"
 
 if [[ ${PV} == 9999 ]]; then
 	inherit git-r3
-	EGIT_REPO_URI="https://github.com/emersion/${PN}.git"
+	EGIT_REPO_URI="https://git.sr.ht/~emersion/${PN}"
 else
-	SRC_URI="https://github.com/emersion/${PN}/archive/v${PV}.tar.gz -> ${P}.tar.gz"
+	SRC_URI="https://git.sr.ht/~emersion/${PN}/archive/v${PV}.tar.gz -> ${P}.tar.gz"
```

Comment:
This is not fine for old ebuilds, since the tarball from mirrors would be the old one. Changing HOMEPAGE is fine in old ebuilds, but not SRC_URI!

Reply:
> This is not fine for old ebuilds, since the tarball from mirrors would be the old one. Changing HOMEPAGE is fine in old ebuilds, but not SRC_URI!

Thanks! Forgot about the homepage. I'll also update the changelog and bugs-to fields in metadata.xml